services/search/src/search/routes/search.py

```diff
@@ -211,9 +211,13 @@ async def search_endpoint(request: Request) -> Response:
                         extensions_directory,
                     )
                     # no need to do it every time
+                    # TODO: Will be moved to another process in parallel
```

Comment:
I'm not a fan of putting TODOs in the code (I also do it too often, but I think it's a bad idea because it promises something that might never occur). We already have an open issue for it.